tiles-generation/cdk/lambda/task-status-notify.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ * 
+ * Lambda function code for tiles generation ECS task change notification.
+ */
+
+const AWS = require("aws-sdk");
+
+exports.handler = function(event, context, callback) {
+    const sns = new AWS.SNS();
+    
+    const eventText = JSON.stringify(event, null, 2);
+    console.log(eventText);
+
+    let topicArn;
+    try {
+        topicArn = process.env.topicArn;
+    } catch (error){
+        console.error(error, 'Caught error for topicArn');
+    }
+
+    let container;
+    try {
+        container = event.detail.containers[0];
+    } catch (error){
+        console.error(error, "Caught error for container");
+    }
+    
+    let subject;
+    
+    if (container === undefined) {
+        const stoppedReason = event.detail.stoppedReason;
+        subject = `Error: Tile generation container was not found: ${stoppedReason}`;
+        console.log(`Error: Tile generation container was not found: ${stoppedReason}`);
```

Comment:
we can log subject, both the strings are same.

Reply:
will update.